cernopendata/templates/cernopendata_ispy/ispy_help.html

```diff
@@ -22,41 +22,41 @@ <h3>Top-row buttons</h3>
             Each button comes with a tool-tip: hover over the button to know its functionality.
           </p>
           <p>
-          <div class="row">
-            <div class="col-sm-6">
-              <i class="fa fa-folder-open"></i> Open <kbd>.ig</kbd> files from the server.<br/><img class="screen-s"
+          <div class="row ispy-help-row">
+            <div class="col-sm-6 ispy-help-item">
+              <i class="fa fa-folder-open"></i> Open <kbd>.ig</kbd> files from the server.<br/><img class="ispy-help-img"
                                                                                                     src="{{ url_for('static', filename='img/iSpy-Online-graphics/open.gif') }}">
             </div>
-            <div class="col-sm-6">
+            <div class="col-sm-6 ispy-help-item">
               <i class="fa fa-step-backward"></i> and <i class="fa fa-step-forward"></i> Navigate to previous and next
-              events in the <kbd>.ig</kbd> file you loaded.<br/><img class="screen-s"
+              events in the <kbd>.ig</kbd> file you loaded.<br/><img class="ispy-help-img"
                                                                      src="{{ url_for('static', filename='img/iSpy-Online-graphics/prevnext.gif')}}">
             </div>
-            <div class="col-sm-6">
+            <div class="col-sm-6 ispy-help-item">
               <i class="fa fa-home"></i> Returns the event display to its initial orientation. <i
-              class="fa fa-search-plus"></i> and <i class="fa fa-search-minus"></i> Zoom in and out.<br/><img
-              class="screen-s" src="{{ url_for('static', filename='img/iSpy-Online-graphics/start.gif')}}">
+              class="fa fa-search-plus"></i> and <i class="fa fa-search-minus"></i> Zoom in/out.<br/><img
```

Comment:
Please replace also the first "and" with "/", so that plus/minus icons would have a slash between themselves, analogously to the later "in/out" string.

Reply:
Sorry I didn't notice this in your message. Fixed.